src/js/store/module/Recommendations.js

```diff
@@ -0,0 +1,127 @@
+import * as actionsList from '../actions-list'
+import MovieFetcher from '../../service/movie/fetcher'
+import Evaluator from '../../service/movie/evaluator'
+import Filter from '../../service/movie/filter'
+import Sorter from '../../service/movie/sorter'
+
+// Inline here all the mutations / actions constants
+const RECOMMENDATIONS_RETRIEVED = 'RECOMMENDATIONS_RETRIEVED'
+const IS_RECORDING = 'IS_RECORDING'
+
+const getters = {
```

Comment:
I forgot, but that needs to change.
I would put here all the type checks I moved in the view. So the view is not aware of the shape of the `asyncData` data structure. All the `v-if="recommendations.type === 'success'"` would be put here with a more meaningful name.